### PR TITLE
magento/devdocs#6535:  Clarify behavior of plugins in Plugins (Interceptors) topic

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides additional explanations for plugins order execution regarding to https://github.com/magento/devdocs/issues/6535.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

## Links to Magento source code

-  [Call Plugins](https://github.com/magento/magento2/blob/2.3.3/lib/internal/Magento/Framework/Interception/Interceptor.php)

whatsnew
Added execution order examples to [Plugin (Interceptors)](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 1]
```diff
@@ -364,11 +364,31 @@ Your plugin classes has this methods:
...
+    }
+}
+```
+
 The execution will be in this order:
 
 *  `PluginA::beforeDispatch()`
 *  `PluginB::beforeDispatch()`
-*  `PluginB::aroundDispatch()` (Magento calls the first half until `callable`)
+*  `PluginB::aroundDispatch()` (Magento calls the first half after `callable`)
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/plugins.md]: I'm not sure this statement is 100% correct. The around plugin is supposed to be executed before callable and after callable. In the modified example it's being executed two times before callable. 

```
*  `PluginB::aroundDispatch()` (Magento calls the first half after `callable`)
*  `PluginB::aroundDispatch()` (Magento calls the second half after `callable`)
```

Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md]: Good catch, @rogyar !

**fixed**

Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**